### PR TITLE
update adviser-template from adviser

### PR DESCRIPTION
**Pull request by harshad16**
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

## Related Issues and Dependencies

https://github.com/thoth-station/common/issues/882

## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

common module looks for metadata label to schedule workflow.
copied from [adviser](https://raw.githubusercontent.com/thoth-station/adviser/master/openshift/adviser-template.yaml)

### CODE REVIEW

File: adviser/base/openshift-templates/adviser.yaml [hunk 1]
```diff
@@ -2,33 +2,54 @@ apiVersion: template.openshift.io/v1
 kind: Template
 metadata:
   name: adviser
+  annotations:
+    description: "Thoth: Adviser"
+    openshift.io/display-name: "Thoth: Adviser"
+    version: 0.6.4
```
Comment (pacospace) [inline adviser/base/openshift-templates/adviser.yaml:8]: is this going to be updated by Argo? @goern

File: adviser/base/openshift-templates/adviser.yaml [hunk 2]
```diff
@@ -218,3 +271,23 @@ objects:
...
+                      value: "{{workflow.parameters.THOTH_ADVISER_JOB_ID}}"
+                    - name: "THOTH_DOCUMENT_ID"
+                      value: "{{workflow.parameters.THOTH_DOCUMENT_ID}}"
+                    - name: "THOTH_ADVISER_METADATA"
+                      value: "{{workflow.parameters.THOTH_ADVISER_METADATA}}"
+                    - name: origin
+                      value: "{{tasks.trigger-integration.outputs.parameters.origin}}"
+                    - name: source_type
+                      value: "{{tasks.trigger-integration.outputs.parameters.source_type}}"
+                when: "{{tasks.trigger-integration.outputs.parameters.source_type}} == KEBECHET"
```
Comment (pacospace) [inline adviser/base/openshift-templates/adviser.yaml:293]: let's not add those for now

File: adviser/base/openshift-templates/adviser.yaml [hunk 3]
```diff
@@ -133,6 +158,20 @@ objects:
...
+              - key: kafka_ca.crt
+                path: kafka_ca.crt
+        - name: ssh-config
+          secret:
+            secretName: kebechet-secret
+            items:
+              - key: ssh-privatekey
+                path: id_rsa
+                mode: 0600
+
```
Comment (pacospace) [inline adviser/base/openshift-templates/adviser.yaml:174]: sorry let's remove those

File: adviser/base/openshift-templates/adviser.yaml [hunk 4]
```diff
@@ -188,6 +227,20 @@ objects:
...
+                  name: "unresolved-package-handler"
+                  template: "unresolved-package-handler"
+                arguments:
+                  artifacts:
+                    - name: outputdocument
+                      from: "{{tasks.advise.outputs.artifacts.outputdocument}}"
+                  parameters:
+                    - name: THOTH_DOCUMENT_ID
+                      value: "{{workflow.parameters.THOTH_DOCUMENT_ID}}"
+
```
Comment (pacospace) [inline adviser/base/openshift-templates/adviser.yaml:243]: sorry let's remove those, as templates are missing in thoth-application

Comment (harshad16) [reply adviser/base/openshift-templates/adviser.yaml:243]: thanks for pointing these @pacospace 
will update this in new pr.

File: adviser/base/openshift-templates/adviser.yaml (hunk 3 above)
Comment (harshad16) [reply adviser/base/openshift-templates/adviser.yaml:174]: thanks for pointing these @pacospace

File: adviser/base/openshift-templates/adviser.yaml (hunk 2 above)
Comment (harshad16) [reply adviser/base/openshift-templates/adviser.yaml:293]: thanks for pointing these @pacospace

**Pull request merged**